prusa/link/printer_adapter/file_printer.py

```diff
@@ -358,13 +361,31 @@ def stop_print(self):
             self.data.printing = False
             self.data.paused = False
 
+    def get_bed_temperture(self, file_path):
```

Comment:
```suggestion
    def get_bed_temperature(self, file_path):
```

Reply:
Thank you for finding this typo. Not really sure if this version is going to make it in, it might happen that firmware will make these variable waits not necessary